### ISSUE TITLE
egs2/aishell2 recipe does not converge

### ISSUE THREAD

**Issue by helloworld02202** [espnet/espnet#4577]
The loss suddenly goes up and never recovers at epoch4, without any errors
It happens occasionally, 1 out of 5 times the same setting can work
Is the the learning rate too high (lr=0.0025) that may cause this to happen?

cuda version - 11.1
torch version - 1.8.1
python version - 3.7.3
espnet code version - v.202207
training configuration - egs2/aishell2/conf/tuning/train_asr_conformer.yaml
ngpu = 4

training log

> 2022-08-16 14:00:59,903 (trainer:710) INFO: 4epoch: train:3633-4540batch: iter_time=2.295e-04, forward_time=0.132, loss_ctc=8.908, loss_att=6.127, acc=0.879, loss=6.961, backward_time=0.380, optim_step_time=0.049, optim0_lr0=0.001, train_time=2.409
2022-08-16 14:10:05,469 (trainer:710) INFO: 4epoch: train:4541-5448batch: iter_time=2.382e-04, forward_time=0.133, loss_ctc=8.282, loss_att=5.757, acc=0.883, loss=6.515, backward_time=0.380, optim_step_time=0.051, optim0_lr0=0.001, train_time=2.403
**2022-08-16 14:19:12,659 (trainer:710) INFO: 4epoch: train:5449-6356batch: iter_time=2.297e-04, forward_time=0.133, loss_ctc=73.577, loss_att=38.514, acc=0.347, loss=49.033, backward_time=0.381, optim_step_time=0.049, optim0_lr0=0.001, train_time=2.410**
2022-08-16 14:28:19,264 (trainer:710) INFO: 4epoch: train:6357-7264batch: iter_time=2.323e-04, forward_time=0.133, loss_ctc=74.890, loss_att=38.079, acc=0.359, loss=49.123, backward_time=0.381, optim_step_time=0.051, optim0_lr0=0.001, train_time=2.407
2022-08-16 14:37:23,823 (trainer:710) INFO: 4epoch: train:7265-8172batch: iter_time=2.310e-04, forward_time=0.132, loss_ctc=73.181, loss_att=36.488, acc=0.373, loss=47.496, backward_time=0.380, optim_step_time=0.050, optim0_lr0=0.001, train_time=2.399

**Comment by hardikkamboj** [comment]
@helloworld02202 , 

Can you please tell me what to pass in db.sh for AISHELL2. I have the data, and have prepared it in Kaldi format. But, not sure what to pass in db.sh. 

I have ran the local/prepare_data.sh, which made spk2utt, text, utt2spk, wav.scp . But, when running local/data.sh or run.sh, I get error in line 7 db.sh (where we have to pass the directory of the corpus). 

When I set AISHELL2="data/train", it gives error like "it is a directory", when I pass AISHELL2="data/train/wav.scp", it gives Permission Error.

**Comment by sw005320** [comment]
Please set the path of the original AISHELL2 data by referring to the CMU/JHU cases (https://github.com/espnet/espnet/blob/master/egs2/TEMPLATE/asr1/db.sh#L173-L318).
Or if you could run the data preparation part successfully, you can skip it by using `run.sh --stage 2`
